src/tabs/led_strip.html

```diff
@@ -88,6 +88,20 @@
                 <div class="checkbox rainbowOverlay">
                     <input type="checkbox" name="Rainbow" class="toggle function-y" />
                     <label> <span i18n="ledStripRainbowOverlay"></span></label>
+                    <div class="rainbowSlidersDiv">
+                        <span i18n="ledStripRainbowDeltaSliderTitle"></span>
+                        <div class="rainbowDeltaSlider">
+                            <input type="range" min="0" max="359"/>
+                            <label></label>
+                            <div class="helpicon cf_tip" i18n_title="ledStripRainbowDeltaSliderHelp"></div>
+                        </div>
+                        <span i18n="ledStripRainbowFreqSliderTitle"></span>
+                        <div class="rainbowFreqSlider">
+                            <input type="range" min="1" max="200"/>
+                            <label></label>
```

Comment:
Empty label

Reply:
Empty on purpose. It will show the slider value and on load it request it from the FC and display in the label.